code/game/gamemodes/clock_cult/clock_scripture.dm

```diff
@@ -1135,15 +1136,15 @@ Judgement: 10 servants, 100 CV, and any existing AIs are converted or destroyed
 	<span class='userdanger'>You feel limitless power surging through you!</span>")
 	playsound(invoker, 'sound/magic/clockwork/invoke_general.ogg', 50, 0)
 	playsound(invoker, 'sound/magic/lightning_chargeup.ogg', 100, 0)
-	animate(invoker, color = list(rgb(255, 255, 255), rgb(255, 255, 255), rgb(255, 255, 255), rgb(0,0,0)), time = 88) //Gradual advancement to extreme brightness
+	animate(invoker, color = "#FFFF00", time = 88) //Gradual advancement to extreme brightness
```

Comment:
"#FFFFFF" would appear completely transparent though, that's why I opted for yellow.
I'm indeed using the beta version, I guess I'll revert my code.